tests/scipy_stats_test.py

```diff
@@ -889,5 +890,84 @@ def evaluate_kde(kde, x):
     tree_util.tree_map(lambda a, b: self.assertAllClose(a, b), kde, kde2)
     self.assertAllClose(evaluate_kde(kde, x), kde.evaluate(x))
 
+  @parameterized.named_parameters(
+    jtu.cases_from_list(
+      {
+        "testcase_name":
+          f"""{x_shape};{x_dtype};{axis};{nan_policy};{contains_nans} """,
```

Comment:
A couple things here:
- We generally use underscores rather than semicolons in test names.
- please use the `format_shape_dtype_string` utility for the shape and type
- please add parameter names to the test string; e.g. `_axis={axis}_`
- Use single quotes for a one-line string

Reply:
Word, thanks